pkg/crc/cluster/cluster.go

```diff
@@ -223,3 +223,15 @@ func DeleteOpenshiftApiServerPods(oc oc.OcConfig) error {
 
 	return errors.RetryAfter(60, deleteOpenshiftApiserverPods, time.Second)
 }
+
+func CheckProxyIsPartOfAuthOperator(oc oc.OcConfig, proxy *network.ProxyConfig) (bool, error) {
+	cmdArgs := []string{"set", "env", "deployment", "redhat-operators", "--list", "-n", "openshift-marketplace"}
+	out, _, err := oc.RunOcCommandPrivate(cmdArgs...)
+	if err != nil {
+		return false, err
+	}
+	if strings.Contains(out, proxy.HttpsProxy) {
```

Comment:
What if only HttpProxy is set?

Reply:
Ah yes I have to put an or operator here, had in mind but just missed when putting the code :(